task.py

```diff
@@ -294,12 +294,11 @@ def node_name(self) -> str:
     def node_name_sanitized(self) -> str:
         st = list(self.node.name)
         for i, _ in enumerate(st):
-            if st[i] == '.':
-                st[i] = '-'
+            if st[i] == ".":
+                st[i] = "-"
```

Comment:
let's not do a commit that fixes  earlier commits of the same work.

Every commit individually should be sensibly correct, self-contained and pass basic checks (in this case `black` formatting).